src/gs-details-page.c

```diff
@@ -2448,6 +2448,9 @@ gs_details_page_plugin_status_changed_cb (GsPluginLoader *plugin_loader,
                                           GsPluginStatus status,
                                           GsDetailsPage *self)
 {
+	if (!app)
+		return;
```

Comment:
Maybe you should still set the `self->plugin_status` (last line not shown here).

Reply:
:+1: